c8y-lora-sdk/src/main/java/com/cumulocity/lpwan/codec/sdk/CodecMicroservice.java

```diff
@@ -0,0 +1,21 @@
+package com.cumulocity.lpwan.codec.sdk;
+
+import com.cumulocity.lpwan.codec.sdk.model.DeviceInfo;
+import org.springframework.stereotype.Component;
+
+import java.util.Set;
+
+/**
+ * This abstract class will be implemented to provide a set of unique supported devices.
```

Comment:
<h1>Codec Microservice</h1>
CodecMicroservice should be implemented by the microservice implementer to advertise the devices it supports.